### ISSUE TITLE
Any possible shader improvements?

### ISSUE THREAD

**Issue by WoodsSilent05** [ShadowMario/FNF-PsychEngine#15661]
### What is your question?

This may not be the right place/label to put this in, but,

I'm currently working on a modified version of the engine that fixes some things and adds some features that I find useful (ex: hiding parts of the text on the Hud), and I know shaders can be a bit funky, they can cause lag even though they aren't complex sometimes, so I want to find ways to improve the shader function in source, but shaders are something out of my league, so I'm asking for help here. If yall have anything that can improve the shader function (in source), that would be very appreciated! :)

TLDR: Give shader function improvement ideas/the code for the improvements :)

**Comment by SantiagoCalebe** [comment]
`I don't think Shadow will optmize it since you're using a modified version.`

**Comment by WoodsSilent05** [comment]
lol forgot to reply, but I wasnt asking shadow, i was asking the community, but ill just post this elsewhere.

**Comment by SantiagoCalebe** [comment]
Lil' bro can't take a good reply

**Comment by WoodsSilent05** [comment]
mkay
